src/components/App.jsx

```diff
@@ -1,43 +1,50 @@
-import autoBind from 'react-autobind';
-import React from 'react'
-import cloneDeep from 'lodash.clonedeep'
-import clamp from 'lodash.clamp'
-import buffer from 'buffer'
-import get from 'lodash.get'
-import {unset} from 'lodash'
-import {arrayMoveMutable} from 'array-move'
-import url from 'url'
+import { arrayMoveMutable } from "array-move";
```

Comment:
Is it possible to revert the lint change and simply add the relevant part?
I'm having a hard time finding the relevant diff and I prefer to run a linter in a separate PR.
See: 
- #839

Reply:
As we deleted the original branch, not sure what you mean by reverting.
Do you mean develop again?

Reply:
Taking the original `App.jsx` file and only adding the relevant part without all the prettier changes.

Reply:
So, rework...
OK. But first let's see that this PR is even built